Makefile

```diff
@@ -27,6 +27,10 @@ EXTENSION = .c
 m8c: $(OBJ)
 	$(CC) -o $@ $^ $(local_CFLAGS) $(INCLUDES)
 
+libusb: INCLUDES = $(shell pkg-config --libs sdl2 libusb)
```

Comment:
On my system pkg-config wanted libusb-1.0 as the parameter ... but I'm not sure if it's just my distro (Fedora 37)?

Reply:
Yeah, same actually on a mac :)